File: ISP63_BackEnd2024/angular/src/app/modelo/mesaExamen/mesaExamenListGetDTO.ts
export interface MesaExamenListGetDTO {
    id: number;             // id mesa
    llamado: string;
    fechayHora: Date;        // LocalDateTime en Java se convierte a Date en TypeScript
    anio: number;
    asignatura: string;
    carrera: string;
    tribunal: string;
}

File: ISP63_BackEnd2024/src/main/java/com/backend/ISP63/dto/MesaExamenListGetDTO.java
package com.backend.ISP63.dto;


import com.backend.ISP63.enums.LlamadoTurno;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MesaExamenListGetDTO {

    private Long id; //id mesa
    private String llamado;
    private String fechayHora;

    private int anio;
    private String asignatura;
    private int idCarrera;
    private String carrera;

    private String tribunal;
}
